packages/website/prettier.config.cjs

```diff
@@ -0,0 +1,16 @@
+/** @type {import('prettier').Config} */
+module.exports = {
+  plugins: ['prettier-plugin-tailwindcss'],
+  arrowParens: 'always',
+  bracketSpacing: true,
+  endOfLine: 'lf',
+  jsxSingleQuote: false,
+  printWidth: 100,
+  quoteProps: 'consistent',
+  semi: true,
+  singleQuote: true,
+  tabWidth: 2,
+  trailingComma: 'all',
+  tailwindConfig: './tailwind.config.js',
+  tailwindFunctions: ['cx'],
+};
```

Comment:
I think the prettier config should be the same across the monorepo. It may require to install prettier-plugin-tailwindcss in root, but that can't be helped, I guess.